cluster/gce/gci/configure-helper.sh

```diff
@@ -708,12 +708,15 @@ function start-kube-addons {
     metrics_memory="${base_metrics_memory}"
     base_eventer_memory="200Mi"
     eventer_memory="${base_eventer_memory}"
+    nanny_memory="100Mi"
```

Comment:
I think it shouldn't.

please change to 90 (as we don't want to increase resources for small clusters)